MC/bin/o2dpg_sim_workflow.py

```diff
@@ -980,8 +988,11 @@ def getDigiTaskName(det):
                               + putConfigValuesNew(["ITSClustererParam",
                                                     'TPCGasParam',
                                                     'ITSCATrackerParam',
-                                                    'MFTClustererParam'])                         \
-                              + " --track-sources " + anchorConfig.get("o2-tof-matcher-workflow-options",{}).get("track-sources",toftracksrcdefault) + (' --combine-devices','')[args.no_combine_dpl_devices]
+                                                    'MFTClustererParam',
+                                                    'GPU_recp_tpc',
```

Comment:
Hello @sawenzel , is there a typo here? Should it not be "GPU_reco_tpc" instead of "GPU_recp_tpc"?